Sources/AardvarkMailUI/ARKEmailBugReporter.m

```diff
@@ -472,7 +491,6 @@ - (void)_showEmailComposeWindow;
     [self.mailComposeViewController beginAppearanceTransition:YES animated:YES];
     
     self.emailComposeWindow.rootViewController = self.mailComposeViewController;
-    [self.emailComposeWindow addSubview:self.mailComposeViewController.view];
```

Comment:
With this line here, we get a warning in the console:
```
[Window] Manually adding the rootViewController's view to the view hierarchy is no longer supported. Please allow UIWindow to add the rootViewController's view to the view hierarchy itself.
```
This lines up with the documentation for [UIWindow.rootViewController](https://developer.apple.com/documentation/uikit/uiwindow/1621581-rootviewcontroller):
```
Assigning a view controller to this property (either programmatically or using Interface Builder) installs the view controller’s view as the content view of the window.
```
I tested with this line removed on iOS 14, both in an app with and without a scene delegate, and it worked fine. I'll try it tomorrow on a device running an older version.

Reply:
This works on my device running iOS 12.4.8